pkg/app/args.go

```diff
@@ -31,13 +30,8 @@ func CreateArgs() (Args, error) {
 func validateArgs(args *Args) error {
 	var err error
 	if args.ConfigFile == "" {
```

Comment:
We have a default for the argument, this should never error out, I would rather remove this to avoid confusion IMO.